src/test/java/com/vanniktech/dependency/graph/generator/DependencyGraphGeneratorLinkTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.vanniktech.dependency.graph.generator
+
+import com.vanniktech.dependency.graph.generator.DependencyContainer.Project
+import com.vanniktech.dependency.graph.generator.DependencyContainer.ResolvedDependency
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorExtension.Generator.Companion.ALL
+import guru.nidi.graphviz.attribute.Color.BLUE
+import guru.nidi.graphviz.attribute.Style.DASHED
+import guru.nidi.graphviz.model.Link
+import org.gradle.api.internal.project.DefaultProject
+import org.gradle.api.plugins.JavaLibraryPlugin
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class DependencyGraphGeneratorLinkTest {
+
+  private lateinit var project: DefaultProject
+
+  @Before
+  fun setUp() {
+    project = ProjectBuilder.builder().withName("root").build() as DefaultProject
+    project.plugins.apply(JavaLibraryPlugin::class.java)
+    project.repositories.run { add(mavenCentral()) }
+    project.dependencies.add("api", "org.jetbrains.kotlin:kotlin-stdlib:1.2.30")
+    project.dependencies.add("implementation", "io.reactivex.rxjava2:rxjava:2.1.10")
+  }
+
+  @Test
+  fun customizeLinkBasedOnFromAndTo() {
+    val blueDashedLinkForRootToRxJava = { link: Link, from: DependencyContainer, to: DependencyContainer ->
+      if ((from as? Project)?.project == project && (to as? ResolvedDependency)?.moduleName == "rxjava") link.attrs()
+        .add(BLUE, DASHED)
+      else link
+    }
+    assertEquals(
+      // language=DOT
```

Comment:
All other files have it like this, does casing mater?

```suggestion
      // language=dot
```

Reply:
No it does not, at least for this language